### DISCUSSION TITLE
How to disable column-statistics ? on shared server ?

### DISCUSSION THREAD

**Discussion by iamsagarmaheta** [MySqlBackupNET/MySqlBackup.Net#109] [Q&A] [answered]
i've been using shared hostinger servers which provides mysql database
since it is not VPN i can not disable column-statistic
and when i try to take backup i am getting error cause server does not support column-statistics
so is there any way to disable it ?
i am unable to take backup

**Comment by adriancs2** [comment, marked as answer]
ExportInfo.GetTotalRowsMode - enum BlobDataExportMode

default value: InformationSchema
Gets or Sets a value indicates the method of how the total rows value is being obtained.
This function is useful if you are developing a progress bar
. InformationSchema = Fast, but approximate value;
SelectCount = Slow but accurate;
Skip = Skip obtaining total rows. Use this option if you are not doing any progress report.

Select "Skip"
